### PR TITLE
Feature/lgcs 661

### PR DESCRIPTION
**Pull request by h0usemaintenance**
User registration and login functionality.

### CODE REVIEW

**Review by joel-stobart — DISMISSED** [review]
I think it's good to get this up. We can address the changes with follow on tickets

File: CyberHealth/CyberHealth/settings.py [hunk 1]
```diff
@@ -142,8 +145,13 @@
     os.path.join(STATIC_URL, 'dist'),
 ]
 
-BASICAUTH_USERS = {'CyberHealth': 'cyber123'}
-BASICAUTH_DISABLE = env('BASICAUTH_DISABLE', default=False)
+CRISPY_TEMPLATE_PACK = 'bootstrap4'
```
[the remark this answers is not in the document]
Comment (adamgparsons) [reply CyberHealth/CyberHealth/settings.py:148]: I can remove it in the future styling branches if that helps

**Review by joel-stobart — COMMENTED** [review]
A couple of really small things to consider. Happy to merge in general

File: CyberHealth/CyberHealth/settings.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (joel-stobart) [reply CyberHealth/CyberHealth/settings.py:149]: Perfect :)

**Review by joel-stobart — APPROVED** [review]

**Pull request merged**